app/api/__init__.py

```diff
@@ -469,7 +469,7 @@
 
 # discount codes
 api.route(DiscountCodeListPost, 'discount_code_list_post', '/discount-codes')
-api.route(DiscountCodeList, 'discount_code_list', '/events/<int:event_id>/discount-codes',
+api.route(DiscountCodeList, 'discount_code_list', '/events/<int:event_id>/ticket-discount-codes',
```

Comment:
You are using `ticket-discount-codes` and `event-discount-codes` in docs and test but I don't see `event-discount-codes` here? Also, did you open an issue to accompany these changes on FE?

Reply:
@shreyanshdwivedi I got confused. The Url to create ticket discount code is also wrong here. I will open a separate PR for that. And correct the changes here.

Reply:
@shreyanshdwivedi Done. Please review now.